### ISSUE TITLE
Mint update (`mintupdate`) with NVIDIA breaks Cinnamon

### ISSUE THREAD

**Issue by WChargin** [linuxmint/Cinnamon#1949]
**tl;dr:** Running `mintupdate` with a GTX 680 kills Cinnamon/Muffin.

(I've written [a Unix&Linux StackExchange question][1] on this topic with [an answer][2].)

From my original post:

> I have Linux Mint 14 64-bit Cinnamon on a desktop with an Intel GTX 680 graphics card. I downloaded and installed the drivers from the NVIDIA website (installed on TTY1 with sudo service mdm stop). The drivers work and the GPU is fully functional.
>
> However, after running the update manager (because it's a fresh install), the Cinnamon environment becomes severely broken. The title bars are missing and so is the bottom panel. Alt-F2 doesn't work either.
>
> I can get the title bars back with sudo metacity --replace but the panels are still gone.

The answer, as given by [Moataz Elmasry][3], is to use

    sudo apt-get update
    sudo apt-get upgrade

instead of `mintupdate`. However, I don't understand how this is different or why the problem occurs. I'd be grateful for a patch.

 [1]: http://unix.stackexchange.com/questions/72815/mint-update-with-nvidia-breaks-cinnamon/
 [2]: http://unix.stackexchange.com/a/73708/37380
 [3]: http://unix.stackexchange.com/users/37889/moataz-elmasry

**Comment by JosephMcc** [comment]
Closing due to inactivity.
